src/transformers/models/bart/modeling_bart.py

```diff
@@ -410,11 +368,10 @@ def forward(
             encoder_attention_mask (:obj:`torch.FloatTensor`): encoder attention mask of size
                 `(batch, 1, tgt_len, src_len)` where padding elements are indicated by very large negative values.
             past_key_value (:obj:`Tuple(torch.FloatTensor)`): cached past key and value projection states
-            output_attentions (:obj:`bool`): Whether the base model outputs attentions. This requires the attentions tensor to be reshaped in this function.
+            output_attentions (:obj:`bool`): Whether the base model outputs attentions.
+                This requires the attentions tensor to be reshaped in this function.
```

Comment:
I don't understand what this means

Reply:
yeah me neither :D Leaving this open as it was probably put in the seq2seq templates